backend/routes/add_repo_route.py

```
@@ -15,22 +14,33 @@ async def list_installation_repos(request: Request):
+        return JSONResponse({"error":"No installation ID"}, status_code=400)
     
     try:
-      branches = await get_repo_branches(installation_id,repo_name)
-      return JSONResponse({"Branches":branches})
-    except Exception as e:
-        print(f"[ERROR] Failed to get repos from installation: {e}")
+        gh = await GitHubMCP.for_installation(installation_id)
+        async with gh:
+            branches = await gh.list_branches(owner,repo_name)
+        return JSONResponse({"Branches": branches})
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)